Project.toml

```diff
@@ -14,9 +13,7 @@ Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 
 [compat]
 ChainRulesCore = "0.9.21"
-ChainRulesTestUtils = "0.5.5"
 Compat = "3"
-FiniteDifferences = "0.11.4"
```

Comment:
these lines should not be removed.
They still function to bind the version of things listed in `[extras]`

Reply:
>  They still function to bind the version of things listed in [extras]

TIL. Pkg documentation is really lacking some bits on test dependencies.

Reply:
done